### DISCUSSION TITLE
关于PPOCRLabel在标注表格结构识别任务时的问题

### DISCUSSION THREAD

**Discussion by Arlen-yuzu** [PaddlePaddle/PaddleOCR#12710] [Q&A]
在使用PPOCRLabel进行表格结构识别标注时，生成的gt.txt文件中的HTML markup格式的token标注应该是PPOCRLabel内置的tableocr模型识别得到的，但是经常会发生HTML markup序列识别出错的情况，在弹出的excel里可以看出识别结构结果是不对的，请问有无方法对识别后的表格结构进行调整。有关cell bbox目前是可以调整的，但是生成的HTML markup出错就难以调整了，如果难以调整表格结构的话，表格识别这个标注功能感觉就没法用了。

**Comment by gl94** [comment]
修改excel成对的,html序列是根据excel生成的

**Comment by SWHL** [comment]
因PPOCRLabel已经搬家了，后续有问题请移步[PPOCRLabel](https://github.com/PFCCLab/PPOCRLabel)提问哈！
此issue先行关闭了。
